CONTRIBUTING.md

```diff
@@ -545,194 +740,3 @@ Now we can run the python interpreter and pass the arguments (the python file to
     Target 0: (python) stopped.
 
 After this, you can step through the code and continue with your debug session as always.
-
-
-## Style guide
```

Comment:
Can you restore this info? 
- We will eventually (sooner than later) have a style guide.
- Also, I think it's important for contributors to take into account that we still need to be compatible with all supported platforms (as there's a lot of native code).
- Informing about the Good First Contributions program is interesting for newcomers.
- The guide on how to create PRs is useful as well.
- It's important to show how a good commit message should look like.

Reply:
See the first section of the updated contributing guide. It says read the qiskit-wide contributing guide: https://qiskit.org/documentation/contributing_to_qiskit.html which contains all this information.

The only thing missing there from your list is the cross platform compatibility which I will add back.

Reply:
Done in: https://github.com/Qiskit/qiskit-aer/pull/642/commits/bc56e1ca0df2915196b37576686b4079e4912e93

Reply:
Thanks Matt!